crates/typed-store/src/rocks/mod.rs

```diff
@@ -1831,7 +1831,7 @@ pub fn base_db_options() -> DBOptions {
     );
     // According to docs, we almost certainly want to set this to number of cores to not be bottlenecked
     // by rocksdb
-    opt.increase_parallelism((num_cpus::get() as i32) / 8);
+    opt.increase_parallelism(4);
```

Comment:
why this change?

Reply:
This increases the size of the threadpool for background jobs i.e. compaction and flushes. Increasing it seems to have helped compaction keep up with the write rate.

Reply:
Yes but it # cores is > 32, then the number is actually decreased from what it was before because its always pegged at 4.
So why not something like
```
if (num_cpus::get() as i32) < 32 {
    4
} else {
    ((num_cpus::get() as i32) / 8)
};
```

Reply:
I think we are likely ok with 4 now that we've tested it at high throughput(default is 2). It seems it is safer to tweak the size of this threadpool manually (after validating in benches) while understanding the perf implications than just based on the num cpus.

Reply:
Seems reasonable